components/app.css

```diff
@@ -16,6 +16,10 @@ html {
   outline: none;
 }
 
+.table {
+  display: inline;
+}
+
```

Comment:
Now all tables are `display: inline;`. That is not good. This should only be the class is used in a code block.

Reply:
Now it only targeting table for toml lang [68df5R20](https://github.com/denoland/deno_website2/pull/1164/files#diff-243e12fe4a5aeeaa414c79fdfb968df5R20)